lib/cluster/membership.c

```diff
@@ -57,6 +57,23 @@ unsigned long long crm_peer_seq = 0;
 gboolean crm_have_quorum = FALSE;
 static gboolean crm_autoreap  = TRUE;
 
+// Flag setting and clearing for crm_node_t:flags
+
+#define pcmk__set_peer_flags(peer, flags_to_set) do {                         \
```

Comment:
You could also rename them to get rid of the pcmk__ prefix, since they're not visible outside of this one file.

Reply:
Good point, done.

I also forgot to add crmcluster_private.h to the Makefile.am so that's done too.